spec/controllers/post_controller_spec.rb

```diff
@@ -100,14 +100,23 @@
 
     describe "#show" do
       context "with valid attributes" do
-        it "returns a 200" do
+        it "returns a 200 with post id" do
```

Comment:
Is calling parameter `id` clear enough? Does it indicate the design choice you have made?

Reply:
Good point. I will fix this and make it more clear.